src/UI/Badge.elm

```diff
@@ -88,23 +92,23 @@ toEl cfg (Badge { content } { tone }) =
 -- Internal
 
 
-toneToColors : BadgeTone -> ( Element.Color, TextColor )
-toneToColors tone =
+toneToColor : BadgeTone -> Palette.Color
+toneToColor tone =
     case tone of
         ToneLight ->
-            ( Palette.gray.lighter, Text.colorInverted )
+            Palette.color Palette.toneGray Palette.lumLighter
```

Comment:
I think this is more a Palette refactor than a Text refactor haha :D 

That being said, I would strongly suggest refactoring this `Lum/Brightness`part, I think it causes too much confusion, I would suggest using only `brightness`.

For example, why a `lumDarkest` returns a `Brightness`? we should stick to a single term, either `lum` or `brightness`.

Reply:
![NoooYesss](https://media.giphy.com/media/JnuoU3dg0zD7Qv9yWb/giphy-downsized.gif)